rasa/server.py

```diff
@@ -377,6 +378,34 @@ def validate_request_body(request: Request, error_message: Text) -> None:
         raise ErrorResponse(HTTPStatus.BAD_REQUEST, "BadRequest", error_message)
 
 
+def events_request_format_spec() -> Dict[Text, Any]:
+    """Expected request body schema used for validation."""
+    return EVENTS_SCHEMA
```

Comment:
I followed the example from [action server response validation ](https://github.com/RasaHQ/rasa/blob/7d4164f67f594a441dc951092ad15b8a096d3bca/rasa/core/actions/action.py#L606)and that was using a callable as schema argument, but I've just tried this `validate(events, EVENTS_SCHEMA)` and the tests pass. I can remove this function since `jsonschema` doesn't enforce a type for schema argument.

Reply:
in the action server response validation it also passes **the result** from calling the callable. 
Let's omit this 👍🏻